ci/docker/runtime_functions.sh

```diff
@@ -1544,17 +1480,7 @@ nightly_model_backwards_compat_train() {
     ./tests/nightly/model_backwards_compatibility_check/train_mxnet_legacy_models.sh
 }
 
-# Nightly 'MXNet: The Straight Dope' Single-GPU Tests
-nightly_straight_dope_python2_single_gpu_tests() {
-    set -ex
-    cd /work/mxnet/tests/nightly/straight_dope
-    export PYTHONPATH=/work/mxnet/python/
-    export MXNET_TEST_KERNEL=python2
     export DMLC_LOG_STACK_TRACE_DEPTH=10
```

Comment:
Is the diff just a bit off or did you miss a few parts?

Reply:
I think these lines were added after the creation of this PR and thus it's a merge problem.

Reply:
thanks for noticing this, fixed.